src/js/components/image-loading.jsx

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 
-export function ImageLoading() {
+export function ImageLoading({accentColor} = {}) {
+    const bounceStyle = {};
+
+    if (accentColor) {
+        bounceStyle.backgroundColor = `#${accentColor}`;
+    }
+
     return <div className='image-overlay'>
                <div className='three-bounce spinner'>
-                   <div className='bounce1'></div>
-                   <div className='bounce2'></div>
-                   <div className='bounce3'></div>
+                   <div className='bounce1'
```

Comment:
/shrug you're already in the image-loading component to abstract that kind of things... not sure how "optimal" it's worth going.